frontend/src/app/projects/models/add-git-source/add-git-source.component.spec.ts

```diff
@@ -5,20 +5,20 @@
 
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
-import { CreateCoworkingMethodComponent } from './create-coworking-method.component';
+import { AddGitSourceComponent } from './add-git-source';
 
-xdescribe('CreateCoworkingMethodComponent', () => {
-  let component: CreateCoworkingMethodComponent;
-  let fixture: ComponentFixture<CreateCoworkingMethodComponent>;
+xdescribe('AddGitSourceComponent', () => {
```

Comment:
Is it possible to make this test executable?

Reply:
I will leave it as is for now so we can merge this PR, but we should discuss this next week as it is a more general issue